### PR TITLE
Update mx.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added some channel   + Dailymotion  youtube   twitch  ,they can be fixed when a streamlink proxy will be online

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -1,71 +1,75 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",A+ TV (XHTIT-TDT) (432p) [Offline]
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt2.m3u8
-#EXTINF:-1 tvg-id="XHTVMTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (XHTVM-TDT) (480p) [Offline]
-https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
+#EXTINF:-1 tvg-id="Adn40.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (480p)
```
Comment (freearhey) [inline channels/mx.m3u:4]: It is better to use the codes from the [iptv-org/epg](https://iptv-org.github.io/epg/index.html) where possible to be able to load the program later.

```suggestion
#EXTINF:-1 tvg-id="ADN40.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (480p)
```

Comment (Sphinxroot) [reply channels/mx.m3u:4]: that strange ...  please  edit them  , it wont let me  ,github  tell me im not on the main branch

File: channels/mx.m3u [hunk 2]
```diff
@@ -1,71 +1,75 @@
...
 #EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="Local",Azteca Uno (XHJK-TDT) (1080p) [Offline]
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhjk-tdt.m3u8
-#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-country="MX" tvg-language="" tvg-logo="" group-title="Local",Canal 10 Cancún (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="Local",Canal 10 Cancún (720p) [Not 24/7]
 http://stream2.dynalias.com:1935/live/tvlive1/chunklist_w1211325251.m3u8
 #EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://s3-us-west-1.amazonaws.com/canal10/photos/96800/original.jpg" group-title="Local",Canal 10 Cancún (720p) [Not 24/7]
 http://stream2.dynalias.com:1935/live/tvlive1/playlist.m3u8
-#EXTINF:-1 tvg-id="XEWTTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/12.1.png" group-title="Local",Canal 12 (XETW-TDT) (1080p) [Offline]
-https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xewt-tdt.m3u8
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:19]: @MapGuy11 might tell you why this link is down.

File: channels/mx.m3u [hunk 3]
```diff
@@ -1,71 +1,75 @@
...
 https://video.cdmx.gob.mx/livestream/stream.m3u8
-#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
+#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
 http://204.12.211.210:1935/conectatv/conectatv/playlist.m3u8
-#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
+#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
 #EXTVLCOPT:http-referrer=https://ssh101.com/
 https://tna5.bozztv.com/Conectatvmexico/index.m3u8
+#EXTINF:-1 tvg-id="Forotv.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uNQVYem.png" group-title="News",ForoTV (720p)
+https://live-streams-notusa.televisa.com/channel02-b/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:40]: That one sounds like a backup link to https://live-streams-notusa.televisa.com/channel02/index.m3u8

File: channels/mx.m3u [hunk 4]
```diff
@@ -1,71 +1,65 @@
...
+https://5ca3e84a76d30.streamlock.net/mexiquense/videomexiquense/playlist.m3u8
+#EXTINF:-1 tvg-id="Milenio.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d4/Milenio_Television.png" group-title="",Milenio Televisión (1080p) [Offline]
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCFxHplbcoJK9m70c4VyTIxg/live
+#EXTINF:-1 tvg-id="MonteMaria.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/OzVUVoX.jpg" group-title="Religious",Monte Maria (720p)
+https://rbaca.livestreamingcdn.com/envivo3/smil:live.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="MultimediosGuadalajara.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/0bznKvP.jpg" group-title="",Multimedios Guadalajara (720p)
 https://mdstrm.com/live-stream-playlist/5c54d38ca392a5119bb0aa0d.m3u8
-#EXTINF:-1 tvg-id="MultimediosLaguna.mx" tvg-country="MX" tvg-language="" tvg-logo="" group-title="",Multimedios Laguna (720p)
-http://mdstrm.com/live-stream-playlist_400/57bf686a61ff39e1085d43e1.m3u8
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:58]: Removed because the main live-stream-playlist = adaptive.

File: channels/mx.m3u (hunk 3 above)
Comment (Sphinxroot) [reply channels/mx.m3u:38]: maybe  this one pull out from site

**Review by freearhey — APPROVED** [review]

File: channels/mx.m3u (hunk 2 above)
Comment (MapGuy11) [reply channels/mx.m3u:19]: Yes I moved everything to a new url with Cloudflare and a actual CDN network. I forgot to change the urls for the mexico m3u

Comment (MapGuy11) [reply channels/mx.m3u:19]: If someone could reverse this merge request that would be great

Comment (LaneSh4d0w) [reply channels/mx.m3u:19]: I can copy all the removed links and including them back if you can tell me what's the name of the host

Comment (MapGuy11) [reply channels/mx.m3u:19]: https://cdn87.theus6tv.tk

Comment (LaneSh4d0w) [reply channels/mx.m3u:19]: Noted. Doing the changes.

**Pull request merged**